xlml/utils/name_format.py

```diff
@@ -48,13 +48,14 @@ def generate_tb_file_location(run_name: str, base_output_directory: str) -> str:
 
 
 @task
-def generate_gcs_folder_location(benchmark_id: str) -> str:
+def generate_gcs_folder_location(subfolder: str, benchmark_id: str) -> str:
   """Generates folder location in GCS.
 
   Args:
+    subfolder: Folder name/path for artifacts, such as 'solutions_team/flax'
     benchmark_id: Benchmark id of the test
 
   Returns: GCS folder name with location
   """
   current_datetime = datetime.datetime.now().strftime("%Y-%m-%d-%H-%M-%S")
-  return f"{gcs_bucket.BASE_OUTPUT_DIR}/{benchmark_id}-{current_datetime}/"
+  return f"{gcs_bucket.BASE_OUTPUT_DIR}/{subfolder}/{benchmark_id}-{current_datetime}/"
```

Comment:
NIT: I notice that the subfolder input can be a string contains '/' as you specified in ...config.py. path.join works better regarding the '/' here.

Reply:
+1 to `path.join`

Reply:
sg! I will make the change.